ignite/engine/engine.py

```diff
@@ -614,12 +618,11 @@ def terminate():
                     4 34  |  9
                     2 Engine ended the run at  4 34
 
-        .. versionchanged:: 0.4.10
-            Behaviour changed, for details see https://github.com/pytorch/ignite/issues/2669
-
+        .. versionchanged:: 0.5.1
```

Comment:
We should keep the previous tag and add the new one.
```suggestion
        .. versionchanged:: 0.4.10
            Behaviour changed, for details see https://github.com/pytorch/ignite/issues/2669

        .. versionchanged:: 0.5.2
            Added `skip_event_completed` flag
```